### PR TITLE
changes suggested by Aaron Raab

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

File: content/device-management-application/viewing-all-devices-bundle/default-columns.md [hunk 1]
```diff
@@ -1,48 +1,63 @@
...
+<tr>
+<td style="text-align:left">Alarms</td>
+<td style="text-align:left">The alarm status of the device, showing number and type of alarms currently unresolved for the device. Only includes alarms for the parent device. See <a href="#alarm-monitoring" class="no-ajaxy">Working with alarms</a> for further information on working with alarms.</td>
+</tr>
+</tbody>
+</table>
+
+For users with global roles the devices list displays the items in a paginated manner. You can select the number of items per page and directly jump to any page. Users with inventory roles see up to 50 items initially. If there are more than 50 devices available more devices are loaded as you scroll down the list.
 
 ### To filter devices
```
Comment (BeateRixen) [inline content/device-management-application/viewing-all-devices-bundle/default-columns.md:62]: This should not be a heading under "Default columns" but at the same level with it. The same for the following heading.

File: content/certified-devices.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+icon: "c8y-icon c8y-icon-device"
+type: root
+layout: redirect
+bundle: certified-devices
+section:
+  - device_management
+weight: 70
+---
+
+Browse Cumulocity IoT certified IoT devices in the Device Catalog of the Cumulocity IoT Connect Device Partner Portal.
```
[the remark this answers is not in the document]
Comment (rssag) [reply content/certified-devices.md:12]: do you mean the tile link?
ticket is here: https://cumulocity.atlassian.net/browse/MTM-54491

File: content/device-management-application/viewing-all-devices-bundle/default-columns.md (hunk 1 above)
Comment (rssag) [reply content/device-management-application/viewing-all-devices-bundle/default-columns.md]: added

File: content/device-management-application/managing-device-repositories.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 weight: 60
-title: Managing device data
+title: Managing device repositories
```
[the remark this answers is not in the document]
Comment (rssag) [reply content/device-management-application/managing-device-repositories.md:3]: communicated, waiting for feedback

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**